### ISSUE TITLE
Narrator doesn't announce the accessible name of list items present below 'History' tab item

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#1462]
Narrator doesn't announce the accessible name of list items present below 'History' tab item.

**Prerequisite**
1. Perform some calculation so that 'History' tab item should have list items.

**Repro steps**
1. Launch 'Calculator'.
2. Home screen will be displayed.
3. Open Narrator by pressing 'Window + Ctrl + Enter' key.
4. Navigate to 'History' tab item and activate it.
5. Navigate to list items present below it.
6. Now verify that Narrator pronounce the accessible name of list items or not.

**Actual Result**
Narrator doesn't announce the accessible name of list items present below 'History' tab item, Narrator only announces the position number of first list item like 1 of 3.

**Expected Result**
Narrator should announce the accessible name of list items present below 'History' tab item, For first list item like "2 x 3 = 6" Narrator should announce complete information as 2 multiply 3 equals to 6 1 of 3.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.
